sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -21,6 +21,12 @@
 - Added the Abstractive Summarization feature and related models: `AbstractSummaryAction`, `AbstractSummaryActionResult`,
  `AbstractSummaryResultCollection`, `AbstractSummaryResult`, `AbstractiveSummary`, and `SummaryContext`. 
   Access the feature through the `beginAnalyzeActions` API.
+- Added the dynamic text classification on documents without needing to train a model. The feature can be used by calling:
+    - `DynamicClassifyDocumentResultCollection dynamicClassificationBatch(Iterable<String> documents, String language, DynamicClassificationOptions options)`    
+    - `Response<DynamicClassifyDocumentResultCollection> dynamicClassificationBatchWithResponse(Iterable<TextDocumentInput> documents, DynamicClassificationOptions options, Context context)`
+    - `Mono<DynamicClassifyDocumentResultCollection> dynamicClassificationBatch(Iterable<String> documents, String language, DynamicClassificationOptions options)`
+    - `Mono<Response<DynamicClassifyDocumentResultCollection>> dynamicClassificationBatchWithResponse(Iterable<TextDocumentInput> documents, DynamicClassificationOptions options)`.
```

Comment:
Does Java normally include all the overloads present in the changelog?

Reply:
There is no standard for it. I could show the max-overload method name only and mention other overload methods.